specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-06-01/managedClusters.json

```diff
@@ -1336,10 +1339,7 @@
         "count": {
           "type": "integer",
           "format": "int32",
-          "maximum": 100,
-          "minimum": 1,
```

Comment:
the min and max still sustains, why also removing them?

Reply:
I initially left them in - but that still generates an "InclusiveMinimum" which requires that the value is between 1 & 100 (which 0 won't be) - hence why I removed these in the second commit ([which removes them from the codegen](https://github.com/Azure/azure-sdk-for-go/pull/6302/commits/3ddeb12b87f3cc24c727bfee37bac69f1272e58b))